core-services/image-mirroring/openshift/GUIDELINES.md

```diff
@@ -0,0 +1,31 @@
+# Overview
+
+The final step of contributing a new image to the release stream is achieved by adding the mirroring
+configuration to the files in this directory.
+
+Currently we also use this step as a final opportunity to ensure we got the image name correct.  That
+is why PRs that touch these files must be approved by a very limited set of individuals.
+
+See below for general guidelines the names should follow.  Ideally we'd move this naming validation
+to earlier in the "new component" process so contributors don't have to go back and rename a bunch
+of things if their name choice needs to be changed at this step.  Fixing that part of the process
+is TBD.
+
+
+# Guidelines
+
+The purpose of these guidelines are to give consistency to our image names, as well as help consumers
+of our images to understand the component/technology/function the image relates to.  This will aid
+them in problem determination and understanding the implications of issues with a particular image.
+
+## Segmentation Ordering
+Image names should follow one the following segmentation orderings:
+
+`$cloudProvider-$component-$optionalModifier` - e.g. aws-ebs-csi-driver-operator
+
+
+`$technology-$component-$optionalModifier` - e.g. sriov-network-config-daemon
```

Comment:
Maybe add `cluster-*` in a follow up.

Reply:
added

Reply:
The same comment about parsing the sample applies here.